js/README.md

```diff
@@ -181,10 +190,10 @@ e.g. `this.setPalette('yourPaletteName);`
     }
   }
 ``` 
-Note: After the new update there is no requirement for a `beginnerMode` check as `BaseBlock` automatically performs that check. 
+Note: After the new update there is no requirement for a `beginnerMode` check as setup() defined in `BaseBlock` automatically performs that check.  
```

Comment:
I think you need to mention that the determination of whether or not a block is beginner mode is determined by a list in turtledefs.js

Reply:
Done